engine/graphics/src/vulkan/graphics_vulkan.cpp

```diff
@@ -3635,6 +3644,27 @@ namespace dmGraphics
         return true;
     }
 
+    static bool VulkanIsAssetHandleValid(HContext _context, HAssetHandle asset_handle)
+    {
+        if (asset_handle == 0)
+        {
+            return false;
+        }
+
+        VulkanContext* context = (VulkanContext*) _context;
+        AssetType type         = GetAssetType(asset_handle);
+
+        if (type == ASSET_TYPE_TEXTURE)
+        {
+            return GetAssetFromContainer<Texture>(context->m_AssetHandleContainer, asset_handle) != 0;
+        }
+        else if (type == ASSET_TYPE_RENDER_TARGET)
+        {
+            return GetAssetFromContainer<RenderTarget>(context->m_AssetHandleContainer, asset_handle) != 0;
+        }
+        return false;
+    }
```

Comment:
we should generalize this to just take a reference to the container, all the adapters will do this identically